common/src/main/java/io/quarkus/ts/openshift/common/injection/TestResource.java

```diff
@@ -14,6 +14,7 @@
  *     <li>{@link io.quarkus.ts.openshift.common.config.Config}</li>
  *     <li>{@link io.quarkus.ts.openshift.common.util.AwaitUtil}</li>
  *     <li>{@link io.quarkus.ts.openshift.common.util.OpenShiftUtil}</li>
+ *     <li>{@link java.net.URL}</li>
```

Comment:
Also add this info to the README (section _Dependency injection_).

Reply:
ok, will do